src/analytics.ts

```diff
@@ -0,0 +1,28 @@
+import Analytics from 'analytics-node';
+
+// Public write key scoped to data source
+const SEGMENT_WRITE_KEY = '5zLPIzVLZYVK40bEzDo8WbYp1omsbEWx'
+interface GitHubActionEventProperties {
+  branch: string; // branch to be deployed using the action
+}
+
+export function track(properties: GitHubActionEventProperties) {
+  if (process.env['DISABLE_ANALYTICS']) {
+    return;
+  }
+
+  const client = new Analytics(SEGMENT_WRITE_KEY);
+
+  try {
+    client.track({
+      event: 'actions-app-deploy',
+      properties,
+      anonymousId: 'anonymous',
```

Comment:
This is wrong as this way we will have a single user with id `anonymous` for all events. Check this out:
https://github.com/contentful/create-contentful-app/pull/1356/files#diff-6540681ca319c66cefe4c76a00d53005729af0d4f4073959701d25ab394a82e2R24